src/Rendering/prePassRendererSceneComponent.ts

```diff
@@ -147,14 +187,15 @@ export class PrePassRendererSceneComponent implements ISceneComponent {
      * context lost for instance.
      */
     public rebuild(): void {
-        // Nothing to do for this component
+        this.scene.disablePrePassRenderer();
+        this.scene.enablePrePassRenderer();

```

Comment:
Maybe that would deserve a comment as we may not understand why `disableXXX` has to be called before `enableXXX`?